rclc_examples/CMakeLists.txt

```
@@ -33,11 +36,14 @@ ament_target_dependencies(example_executor_convenience rcl rclc std_msgs)
 add_executable(example_executor_trigger src/example_executor_trigger.c)
 ament_target_dependencies(example_executor_trigger rcl rclc std_msgs)
 
+add_executable(lifecycle_node src/lifecycle_node.c)
```

```suggestion
add_executable(example_lifecycle_node src/lifecycle_node.c)
```
Propose to use a consistent naming.